entrypoint.sh

```diff
@@ -18,7 +18,9 @@ GITCRYPT_ENABLE="${GITCRYPT_ENABLE:-"false"}"
 GITCRYPT_PRIVATE_KEY="${GITCRYPT_PRIVATE_KEY:-"/secrets/gpg-private.key"}"
 GITCRYPT_SYMMETRIC_KEY="${GITCRYPT_SYMMETRIC_KEY:-"/secrets/symmetric.key"}"
 
-if [[ ! -f /backup/.ssh/id_rsa ]]; then
+if [[ -f /backup/.gitcookies ]]; then
+    git config --global http.cookiefile /backup/.gitcookies
+elif [[ ! -f /backup/.ssh/id_rsa ]]; then
```

Comment:
Please flip the logic and add gitcookies as another `elif` section

Reply:
```suggestion
elif [[ -f /backup/.ssh/id_rsa ]]; then
elif [[ -f /backup/.gitcookies ]]; then
    git config --global http.cookiefile /backup/.gitcookies
else
    git config --global credential.helper '!aws codecommit credential-helper $@'
    git config --global credential.UseHttpPath true
fi
```
This way?